docs/blog/2019-01-04-using-react-context-api-with-gatsby/index.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Using React Context API with Gatsby
+author: Muhammad Muhsin
+date: 2019-01-04
+tags:
+  - "React"
+---
+
+You often feel the unsettling flash of a bright phone screen while relaxing in a dimly lit room. The "dark mode" ends it, by switching background and foreground colors to reduce eye strain. I decided to add this to my boutique web agency [Laccadive IO's](https://laccadive.io/) new Gatsby-based site.
+
+>
```

Comment:
We have a nice Pullquote component that I'd recommend using here.

[Check it out in this post](https://raw.githubusercontent.com/gatsbyjs/gatsby/master/docs/blog/2018-12-04-gatsby-analogy/index.md)